webapp/src/containers/modals/SubscribeModal/filters/repos.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect, useMemo} from 'react';
+
+import pluginConstants from 'pluginConstants';
+import {filterLabelValuePairAll} from 'pluginConstants/common';
+
+import Dropdown from 'components/dropdown';
+
+import usePluginApi from 'hooks/usePluginApi';
+
+import {formLabelValuePairs} from 'utils';
+
+type ReposFilterProps = {
+    handleSelectRepo: (repo: string) => void
+    selectedRepo: string
+} & ReposSubscriptionFiltersRequest
+
+const ReposFilter = ({handleSelectRepo, project, organization, selectedRepo}: ReposFilterProps) => {
+    const {
+        getApiState,
+        makeApiRequestWithCompletionStatus,
+    } = usePluginApi();
+
+    const reposSubscriptionFiltersRequest = useMemo<ReposSubscriptionFiltersRequest>(() => ({
+        organization,
+        project,
+    }), [organization, project]);
+
+    useEffect(() => {
+        makeApiRequestWithCompletionStatus(
+            pluginConstants.pluginApiServiceConfigs.getRepositories.apiServiceName,
+            reposSubscriptionFiltersRequest,
+        );
+    }, [reposSubscriptionFiltersRequest]);
+
+    const {data, isLoading: isGetRepositoriesLoading, isError: isGetRepositoriesError} = getApiState(
```

Comment:
```suggestion
    const {data, isLoading: getRepositoriesLoading, isError: getRepositoriesError} = getApiState(
```

Reply:
I think adding `is` here makes it more readable for a boolean, it's like asking if this is loading which could have either yes or no as a concrete answer